src/core/batch/buffers.ts

```diff
@@ -0,0 +1,38 @@
+import { HexString } from "../../types/Strings";
+
```

Comment:
The `uint8ArrayToHex` is from https://blog.xaymar.com/2020/12/08/fastest-uint8array-to-hex-string-conversion-in-javascript/ and the `hexToUint8Array` is just reversing it.